tools/gnali/.gitignore

```diff
@@ -0,0 +1,7 @@
+*output*
```

Comment:
Please remove this file since we already have one in root of this repo.

Reply:
2a7404f Removed .gitignore